artemis-server/src/main/resources/schema/artemis-configuration.xsd

```diff
@@ -2630,6 +2630,14 @@
             </xsd:annotation>
          </xsd:element>
 
+         <xsd:element name="topology-scanner-attempts" type="xsd:int" default="-1" maxOccurs="1" minOccurs="0">
+            <xsd:annotation>
+               <xsd:documentation>
+                  maximum number of topology scanner attempts, -1 means 'no limits'
```

Comment:
Is there any down side to leaving this scanner running, with a scan period. 0, don't run, > 0 delay X between runs. Is it possible for a subgroup to evolve without a restart, but just by temp network partition. If it is possible, then having the scanner run in the background always by default would make sense.